Sources/Subjects/ReplaySubject.swift

```diff
@@ -0,0 +1,171 @@
+//
+//  ReplaySubject.swift
+//  CombineExt
+//
+//  Created by Jasdev Singh on 13/04/2020.
+//  Copyright © 2020 Combine Community. All rights reserved.
+//
+
+import Combine
+
+/// `ReplaySubject` is a [CurrentValueSubject](https://developer.apple.com/documentation/combine/currentvaluesubject)
+/// that can buffer, well, more than one value! It stores value events, up to its `maxBufferSize` in a
```

Comment:
I do like a more comical writing vibe for blogs, but I'm not sure it fits as much for code documentation. Happy for your thoughts about this 
```suggestion
/// that can buffer more than one value! It stores value events, up to its `maxBufferSize` in a
```

Reply:
Also saying "ReplaySubject is a CurrentValueSubject" might be misleading. WDYT about "ReplaySubject is a subject capable of buffering more than a single value" 

I also think comparing to CurrentValue might be confusing to people who would actually expect to do `.value` on it

Reply:
Agreed, re-worded. 💯